noetic/distribution.yaml

```diff
@@ -4367,6 +4367,13 @@ repositories:
       url: https://github.com/TUC-ProAut/ros_nearfield_map.git
       version: master
     status: maintained
+  neo_local_planner:
+    release:
```

Comment:
Please add a `source` entry pointing to the source repository

Reply:
Updated in #29919